src/core/core/api/task.py

```diff
@@ -29,6 +29,7 @@ def post(self):
         status = data.get("status")
 
         logger.debug(f"Received task result with id {task_id} and status {status}")
+        logger.debug(f"{data=}")
```

Comment:
Printing data without any explanation for every incoming task might not be very helpful here, maybe you could print a subset of information based on a condition